_locales/en/messages.json

```diff
@@ -74,5 +74,9 @@
     "alertUnfollow": {
         "message": "An alert got new unfollows",
         "description": "Alert unfollow"
+    },
+    "alertDeleted": {
+        "message": "The alert {{TITLE}} was deleted",
+        "description": "Alert deleted"
```

Comment:
## French:
- Le alerte {{TITLE}} a été exclu
- Alerte exclu
## Spanish:
- Lo alerta {{TITLE}} ha sido borrado
- Alerta borrado

Ah, also, if you can, replace `Um alerta` in `es` by `Un alerta`. My typo.

Reply:
Thanks!